tasks.py

```diff
@@ -0,0 +1,16 @@
+from invoke import task, run
+"""
+Archivo de tareas para el proyecto
+
+Para ejecutar una tarea usar: invoke <tarea>
+"""
+
+
+@task
+def check(c):
+    """
+    Comprueba si la sintaxis de los ficheros .py son correctos
+    """
+
+    print("Compruebo sintaxis")
+    c.run("pylint src")
```

Comment:
¿Por qué usas pylint? ¿Quién ha dicho que lo uséis? pylint es un comprobador estático de código y buenas prácticas, no un comprobador de sintaxis.